src/client.rs

```diff
@@ -80,12 +81,27 @@ pub fn generate_client_stub(
             if op.leases.values().any(|lease| lease.read && lease.write) {
                 return Err("idempotent operation with read/write lease".into());
             }
+            match &op.reply {
+                syntax::Reply::Result { err, .. }
+                    if matches!(err, syntax::Error::ServerDeath) =>
+                {
+                    return Err(
+                        format!("idempotent operations should not indicate server death: {}", name)
```

Comment:
nitty nit nit:

```suggestion
                        format!("idempotent operations should not indicate server death: {name}")
```

Reply:
That doesn't work with the `rust-toolchain.toml` file in this repo :grimacing:

Reply:
`rust-toolchain.toml` updated to match hubris